src/config/styles.js

```diff
@@ -40,7 +40,7 @@ export const TEXT_SIZE = ({ sm = false, lg = false }) => {
   return `font-size: 16px;`
 }
 
-export const Z_INDEX_TOP = 10
+export const Z_INDEX_TOP = 999
```

Comment:
I also changed this in an upcoming PR, I think to 1000. Anyway it doesn't make a difference so I'll switch it on my branch to 999